compiler/test/test_end_to_end.re

```diff
@@ -664,6 +664,30 @@ let box_tests = [
   t("box_division2", "let b = box(76); b := unbox(b) / 19; ^b", "4"),
 ];
 
+let let_mut_tests = [
```

Comment:
Woah! I wasn't expecting you to write these!

Reply:
😆 I just grepped for box and copy-pasted or replaced everything